docs/painless/painless-guide/painless-datetime.asciidoc

```diff
@@ -661,7 +661,7 @@ preferred as there is no need to parse it for comparison.
 ----
 long now = params['now'];
 ZonedDateTime inputDateTime = doc['input_datetime'];
-long millisDateTime = zdt.toInstant().toEpochMilli();
+long millisDateTime = inputDateTime.toInstant().toEpochMilli();
 long elapsedTime = now - millisDateTime;
 ----
 +
```

Comment:
LGTM. Thanks @ugosan !

As long as we're fixing this example, shouldn't `ZonedDateTime inputDateTime = doc['input_datetime']` be `ZonedDateTime inputDateTime = doc['input_datetime'].value`?

Reply:
You are absolutely right, let me actually review that whole page to make the examples consistent

Reply:
That would be great. Do you want to do that as part of this PR? Or shall we merge this fix, and you can then submit another PR if the review leads to additional changes?